api/src/main/java/org/pmiops/workbench/cohortbuilder/CohortBuilderService.java

```diff
@@ -73,6 +75,10 @@ CriteriaListWithCountResponse findCriteriaByDomain(
 
   Long findDomainCountByStandard(String domain, String term, Boolean standard);
 
+  List<CardCount> findDomainCounts(String term, Boolean standard, List<Domain> domains);
+
+  List<CardCount> findSurveyCounts(String term);
```

Comment:
I would make this one method since they return the same thing List<CardCount>
```suggestion
  List<CardCount> findDomainCounts(String term);
```

Reply:
Move current methods to private and defined a new public method findDomainCounts(term) that encapsulates all controller logic and calls the 2 private methods internally